modules/cluster-autoscaler/variables.tf

```diff
@@ -0,0 +1,37 @@
+variable "cluster_name" {
+  description = "ID of the EKS cluster to deply cluster-autoscaler into."
```

Comment:
We should change the description to EKS cluster name instead of ID

Reply:
It was actually deliberate, since it's called `cluster_id` everywhere in the official `terraform-aws-eks`. But I agree - it should be consistent in our module with either ID or name. I prefer `name` everywhere too.